src/Http/Http.Extensions/test/ProblemDetailsDefaultWriterTest.cs

```diff
@@ -54,6 +54,36 @@ public async Task WriteAsync_Works()
         Assert.Equal(expectedProblem.Instance, problemDetails.Instance);
     }
 
+    [Fact]
+    public async Task WriteAsync_Works_ProperCasing()
+    {
+        // Arrange
+        var writer = GetWriter();
```

Comment:
For this to be a true regression test, don't we need to use a non-web-default options?

```suggestion
        var writer = GetWriter(jsonOptions: new JsonOptions
        {
            SerializerOptions = JsonSerializerOptions.Default
        });
```

Reply:
Good point! I missed that our "default" serialization options are the default web ones. Will fix.